src/app/screens/LNURLChannel/index.tsx

```diff
@@ -74,7 +77,13 @@ function LNURLChannel() {
       }
     } catch (e) {
       console.error(e);
-      if (e instanceof Error) toast.error(`Error: ${e.message}`);
+      if (axios.isAxiosError(e)) {
+        const error =
+          (e.response?.data as { reason?: string })?.reason || e.message;
+        toast.error(`Error: ${error}`);
+      } else if (e instanceof Error) {
+        toast.error(`Error: ${e.message}`);
```

Comment:
If we add this here we should be able to remove https://github.com/getAlby/lightning-browser-extension/pull/3046/files#diff-a082864a4adc3604b70d8b843b52fc353f4debd06482a72388a209bb6701e1f2R47-R49, right?

Reply:
yup true! that's not needed it will eventually be catched and shown in toast. thanks!